### PR TITLE
feat: enable scrolling when added large image

### PR DESCRIPTION
**Pull request by Ir1d**
* Motivation for features / changes

In my case the image is too big I suppose it's 1920x640.
image
After clicking:
image

* Technical description of changes

I let it scroll.

* Screenshots of UI changes

image

But it's not that visually pleasing totally.

image

* Detailed steps to verify changes work correctly (as executed by you)

* Alternate designs / implementations considered

I'm not sure how to make it more beautiful, but the current fix works perfectly for me. Hope it helps.

### CODE REVIEW

**Comment by Ir1d** [comment]
I signed it!

File: tensorboard/plugins/image/tf_image_dashboard/tf-image-loader.html [hunk 1]
```diff
@@ -113,6 +113,7 @@
         height: auto;
         position: relative;
         margin: 0 15px 40px 0;
+        overflow-y: scroll;
```
Comment (stephanwlee) [inline tensorboard/plugins/image/tf_image_dashboard/tf-image-loader.html]: scroll should be changed to `auto`.

Comment (Ir1d) [reply tensorboard/plugins/image/tf_image_dashboard/tf-image-loader.html]: thx! i forgot there's an `auto` :+1:

**Comment by Ir1d** [comment]
btw, do you think it should be `overflow: auto;` instead of x/y ?

File: tensorboard/plugins/image/tf_image_dashboard/tf-image-loader.html [hunk 2]
```diff
@@ -113,6 +113,7 @@
         height: auto;
         position: relative;
         margin: 0 15px 40px 0;
+        overflow-y: auto;
```
Comment (stephanwlee) [inline tensorboard/plugins/image/tf_image_dashboard/tf-image-loader.html]: Actually, thanks for pointing it out, this should be `overflow-x`. In y-axis, the page grows infinitely so there should be no scrollbar. If I am understanding your screenshots correctly, you probably had `overflow-x`.

**Pull request merged**